### PR TITLE
Add cloudfront

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
Add cloud front for the first time

### CODE REVIEW

File: AWS/tests/connectors/s3/test_trigger_s3_cloudfront.py
[the remark this answers is not in the document]
Comment (TOUFIKIzakarya) [reply AWS/tests/connectors/s3/test_trigger_s3_cloudfront.py:1]: Yes of course

File: AWS/tests/connectors/s3/test_trigger_s3_cloudfront.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    connector.module = aws_module
+    connector.configuration = aws_s3_cloudfront_trigger_config
+
+    return connector
+
+
+def test_aws_s3_logs_trigger_parse_data(connector: AwsS3CloudFrontTrigger, test_data_3_1: bytes):
+    assert len(connector._parse_content(test_data_3_1)) == 1
+    assert connector._parse_content(b"") == []
```
Comment (squioc) [inline AWS/tests/connectors/s3/test_trigger_s3_cloudfront.py:87]: Is it expected to parse empty string on each test?

File: AWS/tests/connectors/s3/test_trigger_s3_cloudfront.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    return connector
+
+
+def test_aws_s3_logs_trigger_parse_data(connector: AwsS3CloudFrontTrigger, test_data_3_1: bytes):
+    assert len(connector._parse_content(test_data_3_1)) == 1
+
+
+def test_aws_s3_logs_trigger_parse_data_3_2_2(connector: AwsS3CloudFrontTrigger, test_data_3_2_2: bytes):
+    assert len(connector._parse_content(test_data_3_2_2)) == 2
```
Comment (squioc) [inline AWS/tests/connectors/s3/test_trigger_s3_cloudfront.py:90]: Why did we not test the result of the `_parse_content` for each test?

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**